pkg/template/html/js.go

```diff
@@ -192,8 +195,8 @@ func jsValEscaper(args ...any) string {
 		//     x//* error marshaling y:
 		//          second line of error message */null
 		errStr := err.Error()
+		errStr = string(scriptTagRe.ReplaceAll([]byte(errStr), []byte(`\x3C${1}script`)))
```

Comment:
In the `regexp` package, all functions have two variants: one to operate on `[]byte` and one to operate on `string`.

```suggestion
		errStr = scriptTagRe.ReplaceAllString(errStr, `\x3C${1}script`)
```